README.md

```diff
@@ -54,7 +54,8 @@ A subscription queue should be defined to receive any events raised for the subs
 **Attributes**
 
  - **allow_retry** [Bool] [Optional] [Default=false] This determines if the queue should allow processing failures to be retried.
- - **allow_retry_backoff** [Bool] [Optional] [Default=false] This is used to specify if failed messages that retry should incrementally backoff.
+ - **allow_retry_back_off** [Bool] [Optional] [Default=false] This is used to specify if failed messages that retry should incrementally backoff.
+ - **back_off_weight** [Int] [Optional] [Default=1] Additional multiplier for the timeout backoff. Normally used when `retry_delay` is too small (eg: 30ms) in order to get meaningful backoff values.
```

Comment:
Can we make this `retry_back_off_weight` to keep the naming convention used for `retry_back_off_grace`.

Reply:
sure, just a sec.

Reply:
👍  done